hips/tiles/tests/test_tile.py

```diff
@@ -0,0 +1,34 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.tests.helper import remote_data
+
+from ..tile import HipsTile
+from ..tile_meta import HipsTileMeta
+
+
+class TestHipsTile:
+    def setup(self):
+        self.hips_tile_meta_fits = HipsTileMeta(order=6, ipix=30889, format='fits')
+        self.hips_tile_meta_jpg = HipsTileMeta(order=6, ipix=30889, format='jpg')
+
+    @remote_data
+    def test_fetch_read_write(self):
```

Comment:
The data is always written to a temp directory in case of `pytest`, I think. I say this because during test start, there is a line:

`rootdir: /tmp/hips-test-kp9hkwb4,`

So, it doesn't show up in the project directory. If I were to change this to use use `tmpdir`, I would have to modify `read`, ` write` methods and pass the path along as a parameter. In the case of `classmethod` the path would then have to be stored in `self`.